python/pyarrow/tests/test_pandas.py

```diff
@@ -2164,9 +2172,17 @@ def test_nested_large_list(self):
         s = (pa.array([[[1, 2, 3], [4]], None],
                       type=pa.large_list(pa.large_list(pa.int64())))
              .to_pandas())
-        tm.assert_series_equal(
-            s, pd.Series([[[1, 2, 3], [4]], None], dtype=object),
-            check_names=False)
+
+        # pandas.testing generates a
+        # DeprecationWarning: elementwise comparison failed
+        # numpy.VisibleDeprecationWarning: Creating an ndarray
+        #     from ragged nested sequences ...
+        with warnings.catch_warnings():
+            warnings.simplefilter("ignore", np.VisibleDeprecationWarning)
```

Comment:
Given [our discussion about numpy 1.24](https://github.com/apache/arrow/issues/14759#issuecomment-1330436673) should I remove this warning since it'll be going away?

Reply:
I would still catch/ignore it for now, since we will have test builds for some time that have older numpy